### ISSUE TITLE
Action creators: more control over end action

### ISSUE THREAD

**Issue by jmeas** [jmeas/redux-resource#182]
In #180, @sprjr proposed a way to get a callback when a CRUD action creator is completed. He PR'd a good solution to the problem in #181 .

We chatted about a future enhancement to it where it would pass control over dispatching the end actions to the user. So maybe the callback signature is:

```
(err, res, body, endActionCreator) => {
  // Do stuff, then...
  endActionCreator();
}
```

This would allow you to "build up" successful actions and dispatch them all at once.

---

Pros: more flexible API
Cons: not super easy to do parallel stuff in this system since it's callback-based

**Comment by sprjr** [comment]
Would this still be the recommended path if we end up moving forward with the API in #186? That would allow folks to to do several calls, and then store up some `action.succeeded(res)` values, and then `dispatch(...)` all of those when they are done?

**Comment by jmeas** [comment]
Let's close this out for now. We can revisit this once we have that other lib described in #186 . There will probably be some line we need to figure out when it comes to adding more features to the lib currently known as `resourceful-action-creators` vs recommending people use those new helpers discussed in #186

**Comment by jmeas** [comment]
Resolved in #275
